client/services/FeedbackGeneratorService.js

```diff
@@ -221,6 +221,23 @@ tie.factory('FeedbackGeneratorService', [
         feedback.appendCodeParagraph(errorString);
         return feedback;
       },
+      getPreRequisiteFailureFeedback: function(preReqEvalResult) {
+        var feedback = FeedbackObjectFactory.create(false);
+        var errorMessage =
+          preReqEvalResult.getPreReqErrorMessage();
+        feedback.appendTextParagraph(
+          "Oops!  There is a problem with your code:");
```

Comment:
I wouldn't say "Oops!"; it's a bit too informal. Go for "It looks like there's a problem with your code: "